### PR TITLE
Fix nowplaying  for tv mode

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

**Comment by dmitrylyzo** [comment]
<details>
<summary>To make position slider navigable by keyboard</summary>

```diff
diff -ruN /home/user/tmp/jellyfin-web~gh/dist/components/remotecontrol/remotecontrol.js jellyfin-web~gh_now/components/remotecontrol/remotecontrol.js
--- /home/user/tmp/jellyfin-web~gh/dist/components/remotecontrol/remotecontrol.js	2020-02-15 19:09:27.193879321 +0300
+++ jellyfin-web~gh_now/components/remotecontrol/remotecontrol.js	2020-02-15 20:18:39.146681805 +0300
@@ -1,4 +1,4 @@
-define(["browser", "datetime", "backdrop", "libraryBrowser", "listView", "imageLoader", "playbackManager", "nowPlayingHelper", "events", "connectionManager", "apphost", "globalize", "cardStyle", "emby-itemscontainer", "css!./remotecontrol.css", "emby-ratingbutton"], function (browser, datetime, backdrop, libraryBrowser, listView, imageLoader, playbackManager, nowPlayingHelper, events, connectionManager, appHost, globalize) {
+define(["browser", "datetime", "backdrop", "libraryBrowser", "listView", "imageLoader", "playbackManager", "nowPlayingHelper", "events", "connectionManager", "apphost", "globalize", "layoutManager", "userSettings", "cardStyle", "emby-itemscontainer", "css!./remotecontrol.css", "emby-ratingbutton"], function (browser, datetime, backdrop, libraryBrowser, listView, imageLoader, playbackManager, nowPlayingHelper, events, connectionManager, appHost, globalize, layoutManager, userSettings) {
     "use strict";
 
     function showAudioMenu(context, player, button, item) {
@@ -228,6 +228,11 @@
             buttonVisible(context.querySelector(".btnFastForward"), null != item);
             var positionSlider = context.querySelector(".nowPlayingPositionSlider");
 
+            if (positionSlider && item && item.RunTimeTicks) {
+                positionSlider.setKeyboardSteps(userSettings.skipBackLength() * 1000000 / item.RunTimeTicks,
+                    userSettings.skipForwardLength() * 1000000 / item.RunTimeTicks);
+            }
+
             if (positionSlider && !positionSlider.dragging) {
                 positionSlider.disabled = !playState.CanSeek;
                 var isProgressClear = state.MediaSource && null == state.MediaSource.RunTimeTicks;
@@ -687,6 +692,12 @@
             context.querySelector(".sendMessageForm").addEventListener("submit", onMessageSubmit);
             context.querySelector(".typeTextForm").addEventListener("submit", onSendStringSubmit);
             events.on(playbackManager, "playerchange", onPlayerChange);
+
+            if (layoutManager.tv) {
+                var positionSlider = context.querySelector(".nowPlayingPositionSlider");
+                positionSlider.classList.add("focusable");
+                positionSlider.enableKeyboardDragging();
+            }
         }
 
         function onDialogClosed(e) {
```
</details>

<details>
<summary>To tame focus</summary>

_Still far from perfect. FocusManager needs upgrade_

```diff
diff -ruN /home/user/tmp/jellyfin-web~gh/dist/nowplaying.html jellyfin-web~gh_now/nowplaying.html
--- /home/user/tmp/jellyfin-web~gh/dist/nowplaying.html	2020-02-15 19:09:27.189879383 +0300
+++ jellyfin-web~gh_now/nowplaying.html	2020-02-15 20:00:49.094177303 +0300
@@ -12,7 +12,7 @@
                     <input type="range" is="emby-slider" pin step="1" min="0" max="100" value="0" class="nowPlayingPositionSlider" />
                 </div>
 
-                <div class="nowPlayingInfoButtons">
+                <div class="nowPlayingInfoButtons focuscontainer-x">
                     <button is="paper-icon-button-light" class="btnPreviousTrack btnPlayStateCommand autoSize" title="${ButtonPreviousTrack}">
                         <i class="material-icons">skip_previous</i>
                     </button>
```
</details>

**Comment by grafixeyehero** [comment]
Annotation 2020-02-15 192027

File: src/scripts/librarymenu.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
         html += "</div>";
         html += '<div class="headerRight">';
         html += '<span class="headerSelectedPlayer"></span>';
+        html += '<button is="paper-icon-button-light" class="headerAudioPlayerButton audioPlayerButton headerButton headerButtonRight hide"><i class="material-icons">music_note</i></button>';
```
Comment (dkanada) [inline src/scripts/librarymenu.js]:
```suggestion
        html += '<button is="paper-icon-button-light" class="headerAudioPlayerButton audioPlayerButton headerButton headerButtonRight hide"><i class="material-icons music_note"></i></button>';
```

File: src/scripts/librarymenu.js [hunk 2]
```diff
@@ -82,6 +84,7 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
             if (!layoutManager.tv) {
                 headerCastButton.classList.remove("hide");
             }
+
```
Comment (dkanada) [inline src/scripts/librarymenu.js]: remove this

File: src/scripts/librarymenu.js [hunk 3]
```diff
@@ -140,9 +147,29 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
...
         if (layoutManager.mobile) {
             initHeadRoom(skinHeader);
         }
+        events.on(playbackManager, 'playbackstart', onPlaybackStart);
+        events.on(playbackManager, 'playbackstop', onPlaybackStop);
+    }
+
+    function onPlaybackStart(e) {
+
```
Comment (dkanada) [inline src/scripts/librarymenu.js]: remove newline

File: src/scripts/librarymenu.js [hunk 4]
```diff
@@ -140,9 +147,29 @@ define(["dom", "layoutManager", "inputManager", "connectionManager", "events", "
...
+        if (playbackManager.isPlayingAudio() && layoutManager.tv) {
+            headerAudioPlayerButton.classList.remove("hide");
+        } else {
+            headerAudioPlayerButton.classList.add("hide");
+        }
+    }
+
+    function onPlaybackStop(e, stopInfo) {
+
```
Comment (dkanada) [inline src/scripts/librarymenu.js]: same here

File: src/scripts/site.js [hunk 5]
```diff
@@ -989,10 +989,6 @@ var AppInfo = {};
                 Dashboard.navigate("mypreferencesmenu.html");
             };
 
-            appRouter.showNowPlaying = function () {
-                Dashboard.navigate("nowplaying.html");
-            };
```
Comment (dkanada) [inline src/scripts/site.js]: Where was this used before?

File: src/scripts/librarymenu.js (hunk 1 above)
Comment (dkanada) [reply src/scripts/librarymenu.js]: This will fix WebOS compatibility for any icons with underscores.

**Comment by grafixeyehero** [comment]
for the rest of the icon that have underscores.i will create aother PR

**Pull request merged**